app/models/invalid_identities.rb

```diff
@@ -0,0 +1,20 @@
+class InvalidIdentities
+
```

Comment:
@Stuart-Johnson @amcates The benefit here is for testing purposes. Rake tasks are notoriously difficult to test. You have to load them before the test environment runs. This way you can isolate the testing better, allowing for clearer unit tests. All this is just a PORO (Plain Old Ruby Object), not an ActiveRecord object or anything like that. If it makes it clearer, I could move this file to the lib directory with the others.